platform/ios/app-swift/Sources/ObserverExample.swift

```diff
@@ -10,10 +10,12 @@ class ObserverExampleView: UIViewController, MLNMapViewDelegate {
         super.viewDidLoad()
 
         // #-example-code(actionJournalOptions)
-        MLNSettings.actionJournalOptions.enabled = true
+        let options = MLNMapOptions()
+        options.actionJournalOptions.enabled = true
+        mapView = MLNMapView(frame: view.bounds, options: options)
         // #-end-example-code
 
-        mapView = MLNMapView(frame: view.bounds, styleURL: AMERICANA_STYLE)
+        mapView.styleURL = AMERICANA_STYLE
```

Comment:
```suggestion
        mapView.styleURL = AMERICANA_STYLE
```

Note: this will load this style asynchronously and will cause it to first load the default style.

Maybe we can add the style to MLNMapView as well?

Reply:
To `MLNMapOptions`? That works. I can also disable the default style loading for this specific constructor.

Reply:
Yes I think it makes sense to move towards a paradigm where we use a configuration object. This will avoid further explosion of different initializer permutations.